cmd/kubelet/app/server.go

```diff
@@ -488,13 +493,19 @@ func (s *KubeletServer) kubeconfigClientConfig() (*client.Config, error) {
 // fall back to the default auth (none) without an error.
 // TODO(roberthbailey): Remove support for --auth-path
 func (s *KubeletServer) createClientConfig() (*client.Config, error) {
-	if s.KubeConfig.Provided() && s.AuthPath.Provided() {
-		return nil, fmt.Errorf("cannot specify both --kubeconfig and --auth-path")
-	}
-	if s.KubeConfig.Provided() {
-		return s.kubeconfigClientConfig()
-	} else if s.AuthPath.Provided() {
-		return s.authPathClientConfig(false)
+	authPathFlag := pflag.Lookup(authPathFlagName)
+	kubeconfigFlag := pflag.Lookup(kubeconfigFlagName)
```

Comment:
Shouldn't we be calling lookup on the flagset we added args to? Not sure I like having two objects to keep track of now